lib/test.js

```diff
@@ -70,6 +70,9 @@ Object.keys(assert).forEach(function (el) {
 	};
 });
 
+Test.prototype._capt = assert._capt
+Test.prototype._expr = assert._expr
```

Comment:
@twada Do you have any idea to avoid this? I don't understand much what they are...